src/course-outline/highlights-modal/HighlightsModal.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  ModalDialog,
+  Button,
+  ActionRow,
+  Hyperlink,
+} from '@edx/paragon';
+import { Formik } from 'formik';
+
+import FormikControl from '../../generic/FormikControl';
+import { HIGHLIGHTS_FIELD_MAX_LENGTH } from '../constants';
+import { getHighlightsFormValues } from '../utils';
+import messages from './messages';
+
+const HighlightsModal = ({
+  isOpen,
+  onClose,
+  onSubmit,
+  currentSection,
+  learnMoreVisibilityUrl,
```

Comment:
ref: remove as described in the comment above

Reply:
Done